### ISSUE TITLE
AI.INFO stats 0 calls and duration  for `redisAI.createModelRunner`, `redisAI.createScriptRunner`

### ISSUE THREAD

**Issue by mikhail-vl** [RedisAI/RedisAI#889]
**Is your feature request related to a problem? Please describe.**
It's not clear if it's a bug or a feature request. According to documentation: "The AI.INFO command returns information about the execution of a model or a script. Runtime information is collected each time that AI.MODELEXECUTE or AI.SCRIPTEXECUTE is called." Why not for `redisAI.createModelRunner`, `redisAI.createScriptRunner`?

**Describe the solution you'd like**
AI.INFO should count `redisAI.createModelRunner` and `redisAI.createScriptRunner` calls.

**Describe alternatives you've considered**
N/A

**Additional context**
```
172.26.0.4:30001> keys bert-qa*
1) "bert-qa{06S}"
172.26.0.4:30001> AI.INFO "bert-qa{06S}"
 1) "key"
 2) "bert-qa{06S}"
 3) "type"
 4) "MODEL"
 5) "backend"
 6) "TORCH"
 7) "device"
 8) "CPU"
 9) "tag"
10) ""
11) "duration"
12) (integer) 0
13) "samples"
14) (integer) 0
15) "calls"
16) (integer) 0
17) "errors"
18) (integer) 0
172.26.0.4:30001> 
```

**Comment by AlexMikhalev** [comment]
Redis AI 1.2.5 cpu bionic
Docker build using: 
https://github.com/applied-knowledge-systems/the-pattern-platform/blob/benchmark/conf/Dockerfile 

Steps to reproduce:
```
git clone -b benchmark --recurse-submodules git@github.com:applied-knowledge-systems/the-pattern.git
./bootstrap_benchmark.sh 
redis-cli -c -p 30001 -h 127.0.0.1
```
than output above from @mikhailredis

**Comment by AlexMikhalev** [comment]
Still zeros: 
```
# ai_versions
ai_RedisAI_version:1.2.5
ai_low_level_API_version:1
ai_rdb_version:4

# ai_git
ai_git_sha:746b199e69f272d12aab00c26d31c8e35eb48b29

# ai_load_time_configs
ai_threads_per_queue:1
ai_inter_op_parallelism:0
ai_intra_op_parallelism:0
ai_model_execution_timeout:5000
ai_backend_memory_limit:0

# ai_backends_info
ai_Torch_version:NA

# ai_cpu
ai_self_used_cpu_sys:369.307624
ai_self_used_cpu_user:231.629582
ai_children_used_cpu_sys:5.037731
ai_children_used_cpu_user:22.320745
ai_main_thread_used_cpu_sys:33.307624
ai_main_thread_used_cpu_user:34.629582
ai_queue_CPU_bthread_n1_used_cpu_total:401.920353
```
I thought it was fixed, but no:
```
AI.INFO bert-qa{5Ls}
 1) "key"
 2) "bert-qa{5Ls}"
 3) "type"
 4) "MODEL"
 5) "backend"
 6) "TORCH"
 7) "device"
 8) "CPU"
 9) "tag"
10) ""
11) "duration"
12) (integer) 0
13) "samples"
14) (integer) 0
15) "calls"
16) (integer) 0
17) "errors"
18) (integer) 0
```

**Comment by AlexMikhalev** [comment]
Model called on the shard and returns answer:
```
==> 30006.log <==
64:M 17 Apr 2022 09:44:18.793 * <rg> GEARS: QA cached called from keymiss
64:M 17 Apr 2022 09:44:18.793 * <rg> GEARS: Called with ['bertqa{5Ls}', 'PMC302072.xml:{5Ls}:145', 'Who performs viral transmission among adults']
64:M 17 Apr 2022 09:44:18.793 * <rg> GEARS: Key PMC302072.xml:{5Ls}:145
64:M 17 Apr 2022 09:44:18.793 * <rg> GEARS: Question  Who performs viral transmission among adults
64:M 17 Apr 2022 09:44:18.793 * <rg> GEARS: Shard_id {5Ls}
64:M 17 Apr 2022 09:44:22.007 * <rg> GEARS: Answer her no a pc
64:M 17 Apr 2022 09:44:22.007 * <rg> GEARS: Result her no a pc
64:M 17 Apr 2022 09:44:22.007 * <rg> GEARS: Cache key bertqa{5Ls}_PMC302072.xml:{5Ls}:145_Who performs viral transmission among adults
```
but no reflection in AI.INFO.

**Comment by alonre24** [comment]
Hey @AlexMikhalev,
You should run with Redis AI *edge* cpu bionic docker (instead of 1.2.5). The fix is in our master branch, but we hadn't released yet a new version (1.2.6 version should be out soon)
Can you check and see if it's working as expected with edge docker?

**Comment by AlexMikhalev** [comment]
Confirm fixed:
# ai_git
ai_git_sha:5a66cb397e410f15eb7b1e02a5ebab41afd2cf31
